packages/frontend-api/src/Model/Customer/User/CurrentLoginInfo.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Customer\User;
+
+class CurrentLoginInfo
```

Comment:
```suggestion
class LoginInfo
```

please rename this

Reply:
fixed in https://github.com/shopsys/shopsys/pull/3276/commits/a458824a9e5e3dd087af822a90fa4b9a347c339c